src/MerchantCenter/PriceBenchmarks.php

```diff
@@ -43,18 +234,36 @@ public function update_price_benchmarks(): void {
 			$query->reload_data();
 
 			// Insert new benchmark data.
-			foreach ( $benchmarks as $benchmark ) {
+			foreach ( $benchmarks as $id => $benchmark ) {
+				$performance_data = $this->get_performance_data( $id );
+				if ( ! empty( $performance_data ) ) {
+					// Combine metrics data into the response for the specific product.
+					$benchmark = array_merge( $benchmark, $performance_data );
+				}
+
 				$price_compared_with_benchmark = $this->price_compared_with_benchmark( $benchmark['price_micros'], $benchmark['benchmark_price_micros'] );
+				$effectiveness                 = $this->get_effectiveness( $benchmark['effectiveness'] );
 				$query->insert(
 					[
-						'product_id'                    => $benchmark['offer_id'],
-						'id'                            => $benchmark['id'],
-						'price_micros'                  => $benchmark['price_micros'],
-						'currency_code'                 => $benchmark['currency_code'],
-						'country_code'                  => $benchmark['country_code'],
-						'benchmark_price_micros'        => $benchmark['benchmark_price_micros'],
-						'benchmark_price_currency_code' => $benchmark['benchmark_price_currency_code'],
-						'price_compared_with_benchmark' => $price_compared_with_benchmark,
+						'product_id'                      => $benchmark['id'],
+						'mc_product_id'                   => $benchmark['id'],
```

Comment:
`product_id` should be the WC product ID. `mc_product_id` should be the ID returned from the `PriceCompetitivenessProductView` or `PriceInsightsProductView` queries (e.g., `online:en:US:12345`).